builtin-functions/kphp-light/stdlib/time-functions.txt

```diff
@@ -7,20 +7,23 @@ function mktime ($h ::: int = PHP_INT_MIN, $m ::: int = PHP_INT_MIN, $s ::: int
 
 function date ($format ::: string, $timestamp ::: int = PHP_INT_MIN) ::: string;
 
+function date_default_timezone_get() ::: string;
+
 function date_default_timezone_set ($s ::: string) ::: bool;
 
 function gmdate ($format ::: string, $timestamp ::: int = PHP_INT_MIN) ::: string;
 
 function time() ::: int;
 
+function strtotime ($time ::: string, $timestamp ::: int = PHP_INT_MIN) ::: int | false;
```

Comment:
Can we do argument $timestamp make optional, like it is in [php](https://www.php.net/manual/ru/function.strtotime.php)?

Reply:
I don't know. It requires some investigation whether such change brake existing code

Reply:
I suggest find out it

Reply:
I think it's out of the scope of this PR. It's enough we have the same interface as in `KPHP`